### PR TITLE
OCPBUGS#7659: Change the parameter from --qemu-commandline to --disk

### PR DESCRIPTION
**Pull request by xenolinux**
Change the parameter from --qemu-commandline to --disk

Version(s): 4.9+

Issue:https://issues.redhat.com/browse/OCPBUGS-7659

Link to docs preview: https://56058--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z-kvm.html#installation-user-infra-machines-iso-ibm-z_kvm_installing-ibm-z-kvm

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-ibm-z-kvm-user-infra-machines-iso.adoc [hunk 1]
```diff
@@ -54,6 +54,6 @@ $ virt-install --noautoconsole \
    --disk {disk} \
    --import \
    --network network={network},mac={mac} \
-   --qemu-commandline="-drive \
+   --disk="-drive \
 if=none,id=ignition,format=raw,file={ign_file},readonly=on -device virtio-blk,serial=ignition,drive=ignition"
```
Comment (cfergeau) [inline modules/installation-ibm-z-kvm-user-infra-machines-iso.adoc:58]: The syntax of the parameters after ``--disk`` is different:
```
--disk path={ign_file},format=raw,readonly=on,serial=ignition,startup_policy=optional
```

`-device virtio-blk,serial=ignition,drive=ignition` can be dropped.

Comment (xenolinux) [reply modules/installation-ibm-z-kvm-user-infra-machines-iso.adoc]: @cfergeau Addressed- Does it look good?

Comment (cfergeau) [reply modules/installation-ibm-z-kvm-user-infra-machines-iso.adoc]: This really should be `--disk path={ign_file},format=raw,readonly=on,serial=ignition,startup_policy=optional`, not 
```
   --disk path={ign_file} \
if=none,id=ignition,format=raw,file={ign_file},readonly=on,startup_policy=optional"
```

Comment (xenolinux) [reply modules/installation-ibm-z-kvm-user-infra-machines-iso.adoc]: Addressed

**Review by cfergeau — COMMENTED** [review]
Looks good to me, thanks!

**Comment by stevsmit** [comment]
LGTM.

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Pull request merged**